contracts/core/IPermissions.sol

```diff
@@ -46,4 +50,6 @@ interface IPermissions is IAccessControl, IPermissionsRead {
     function MINTER_ROLE() external view returns (bytes32);
 
     function PCV_CONTROLLER_ROLE() external view returns (bytes32);
+
+    function PCV_GUARD_ROLE() external view returns (bytes32);
```

Comment:
We aren't going to be upgrading the Core contract, so this addition is not needed.